share/rocm/cmake/ROCMInstallTargets.cmake

```diff
@@ -159,16 +159,25 @@ function(rocm_install_targets)
                         NAMELINK_ONLY
             )
         endif()
-        if(T_TYPE MATCHES ".*_LIBRARY" AND ROCM_SYMLINK_LIBS AND NOT ROCM_CREATED_${PROJECT_NAME}_SYMLINK
-            AND NOT CMAKE_HOST_WIN32)
+        if(T_TYPE MATCHES ".*_LIBRARY" AND ROCM_SYMLINK_LIBS AND NOT CMAKE_HOST_WIN32)
+            cmake_policy(SET CMP0087 NEW)
```

Comment:
The use of [`CMAKE_HOST_WIN32`](https://cmake.org/cmake/help/latest/variable/CMAKE_HOST_WIN32.html) rather than [`WIN32`](https://cmake.org/cmake/help/latest/variable/WIN32.html) surprises me. `CMAKE_HOST_WIN32` describes the platform that you're running on and `WIN32` describes the platform that you're building for. I'd expect the latter to be what you care about in this code.

(They'll be identical unless you're cross-compiling, so the distinction is not usually important. Still, it's worth using the variable with the semantics that match your intent.)

Reply:
This was copied from `rocm_install_symlink_subdir`, I believe due to the use of `ln` as opposed to `cmake -E create_symlink`.